backend/app/infrastructure/database/score_delegation.py

```diff
@@ -0,0 +1,23 @@
+from app.infrastructure.database.models import ScoreDelegation
+
+
+def does_delegation_exist(hashed_addr: str) -> bool:
+    delegation = ScoreDelegation.query.filter_by(hashed_addr=hashed_addr).first()
```

Comment:
i think using .exists() will be more performance-oriented?

Reply:
I removed this function, it's not used at all